### ISSUE TITLE
gpt2 pytorch model 344M .bin - Larger models with this codebase

### ISSUE THREAD

**Issue by hypnoai** [graykode/gpt-2-Pytorch#14]
Hi,

I was wondering how to integrate the larger released open.ai models with this code base.

Many thanks,
Vince.

**Comment by rocheston** [comment]
Exactly... This is also my question. How to integrate the newly released OPENAI full model.
Please help. Thanks.
